packages/-ember-data/tests/integration/model-errors-test.ts

```diff
@@ -67,4 +70,24 @@ module('integration/model.errors', function (hooks) {
 
     assert.dom('.error-list__error').doesNotExist();
   });
+
+  test('Uncommitted model can become valid after changing 2 erred fields', async function (this: CurrentTestContext, assert) {
+    this.tag = this.owner.lookup('service:store').createRecord('tag');
+    // @ts-ignore
+    const errors = get(this.tag, 'errors');
```

Comment:
dot notation access should be fine

Reply:
<img width="506" alt="image" src="https://user-images.githubusercontent.com/1536262/180642789-5cb6cb26-ff26-4796-8157-c45198696671.png">